src/standards/policy_rule_issuer.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
These files should not be part of the SRC folder, since they are not source code.

They should be part of the "test" folder!